wordcloud/wordcloud_cli.py

```diff
@@ -20,10 +19,9 @@ def main(args):
         color_func=args.color_func, background_color=args.background_color).generate(args.text)
     image = wordcloud.to_image()
 
-    b = io.BytesIO()
-    image.save(b, format='png')
     with args.imagefile:
-        args.imagefile.write(b.getvalue())
+        out = args.imagefile if sys.version < '3' else args.imagefile.buffer
```

Comment:
So the tests work. I dislike how fragile the test is though - it depends on the call staying the same and inserting the Image object to write stuff out.
It could be replaced with a mock, too - I don't think the situation gets better through that though. Basically everything is reimplemented with mocks at that point, which seems wrong.
A further issue is that only a positive case is tested, while it would be nice to see it failing, too. Oh well.